metadata-ingestion/src/datahub/testing/compare_metadata_json.py

```diff
@@ -0,0 +1,109 @@
+"""Utilities for comparing MCE and MCP files."""
+
+import json
+import logging
+import os
+import pathlib
+import pprint
+import shutil
+import tempfile
+from typing import Any, Dict, List, Sequence, Union
+
+import pytest
+from deepdiff import DeepDiff
+
+from datahub.ingestion.sink.file import write_metadata_file
+from datahub.ingestion.source.file import read_metadata_file
+from datahub.testing.mcp_diff import MCPDiff, get_aspects_by_urn
+
+logger = logging.getLogger(__name__)
+
+MetadataJson = List[Dict[str, Any]]
+
+default_exclude_paths = [
+    r"root\[\d+]\['systemMetadata']\['lastObserved']",
```

Comment:
why do we have `\[` but not `\]` ?

Reply:
It doesn't seem to be necessary -- pycharm initially bugged me about it and I just kept it up for consistency. That being said, I think it does matter if you have an open, unescaped `[` so I should prob escape `]` too just in case